cypress/integration/page.spec.ts

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable max-len */
+import { faker } from '@faker-js/faker';
+import { peopleFromServer } from '../../src/data/people';
+import { Person } from '../../src/types/Person';
+
+const page = {
+  title: () => cy.getByDataCy('title'),
+  searchInput: () => cy.getByDataCy('search-input'),
+  suggestionsList: () => cy.getByDataCy('suggestions-list'),
+  suggestionItem: () => cy.getByDataCy('suggestion-item'),
```

Comment:
`suggestionItems` as I understand it return a collection of items but not just 1 element

So plural form is more natural